src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,115 @@
 package baseball;
+import camp.nextstep.edu.missionutils.Randoms;
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class Application {
+
     public static void main(String[] args) {
-        //TODO: 숫자 야구 게임 구현
+
+        while(true){
+            List<Integer> randomNumber = new ArrayList<>(3);
+
+            //1. 컴퓨터가 랜덤 숫자를 정한다
+            randomNumber = startGame();
+
+            //2. 플레이어가 숫자게임을 시작한다.
+            if(playGame(randomNumber)==1) continue;
+            else break;
```

Comment:
```suggestion
            if (playGame(randomNumber) == 1) {
                continue;
            } else {
                break;
            }
```

* 공백 컨벤션, brace 등 java google style guide에 위배되는 부분들이 있네요.
> https://google.github.io/styleguide/javaguide.html

* code format을 잘 모르겠다면 IntelliJ 또는 Eclipse의 formatting 기능을 활용해보세요.

IntelliJ 단축키는  Ctrl+Alt+L(윈도우), Cmd+Alt+L(맥)
Eclipse 단축키는 Ctrl+Shift+F(윈도우), Cmd+Shift+F(맥)